python/lsst/sphgeom/_yaml.py

```diff
@@ -56,3 +61,53 @@ def region_constructor(loader, node):
 
 for loader in YamlLoaders:
     yaml.add_constructor("lsst.sphgeom.Region", region_constructor, Loader=loader)
+
+
+# Pixelization schemes
+
+def pixel_representer(dumper, data, pixelScheme):
+    """Represent a pixelization in YAML
+
+    Stored as the pixelization level. Use a mapping for more clarity.
```

Comment:
Since these are custom functions, and not PyYAML hooks, please provide a full description of the parameters.

Reply:
The "more clarity" was a note to myself that scalar would be sufficient but a mapping with a level keyword is more explicit. I'll update. I could make the same change for the region serialization as well and use a mapping with a key "encoded" or something and value the hex string rather than a scalar with the hex string directly.